apps/marketplace/pages/OpportunityHistoryPage.js

```diff
@@ -0,0 +1,92 @@
+import React, { Component } from 'react'
+import { connect } from 'react-redux'
+import { withRouter } from 'react-router-dom'
+
+import { loadOpportunityHistory } from 'marketplace/actions/briefActions'
+import OpportunityHistory from 'marketplace/components/Brief/OpportunityHistory'
+import { ErrorBoxComponent } from 'shared/form/ErrorBox'
+import LoadingIndicatorFullPage from 'shared/LoadingIndicatorFullPage/LoadingIndicatorFullPage'
+
+class OpportunityHistoryPage extends Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      dataLoaded: null,
+      loading: false
+    }
+  }
+
+  componentDidMount = () => {
+    if (this.props.match.params.briefId) {
+      this.getOpportunityHistory()
+    }
+  }
+
+  getOpportunityHistory = () => {
```

Comment:
would it be better to pass the briefId into this function?

Reply:
ID now passed to this function.